src/vs/workbench/contrib/terminal/browser/xterm/xtermTerminal.ts

```diff
@@ -800,11 +801,16 @@ class AccessibleBuffer extends DisposableStore {
 			cursorBlinking: configurationService.getValue('terminal.integrated.cursorBlinking'),
 			readOnly: true
 		};
-		this._accessibleBuffer = this._terminal.raw.element!.querySelector('.xterm-accessible-buffer') as HTMLElement;
-		// Prevent the accessible buffer letting mouse events to propogate to xterm.js while it's
-		// visible.
-		this.add(addDisposableListener(this._accessibleBuffer, 'mousedown', e => e.stopImmediatePropagation()));
-		this._accessibleBuffer.tabIndex = -1;
+		this._accessibleBuffer = document.createElement('div');
```

Comment:
We should be able to move this into a contrib now https://github.com/microsoft/vscode/issues/176065